From c6de18839c98d924afb5c6747e17baa1e1a0a946 Mon Sep 17 00:00:00 2001
From: Gary Helmling <garyh@apache.org>
Date: Thu, 14 Jan 2016 09:51:03 -0800
Subject: [PATCH 040/217] HBASE-15111 hbase version should write to stdout

Signed-off-by: Sean Busbey <busbey@cloudera.com>

Conflicts:
	hbase-common/src/main/java/org/apache/hadoop/hbase/util/VersionInfo.java
---
 .../org/apache/hadoop/hbase/util/VersionInfo.java  |    9 ++++++++-
 1 file changed, 8 insertions(+), 1 deletion(-)

diff --git a/hbase-common/src/main/java/org/apache/hadoop/hbase/util/VersionInfo.java b/hbase-common/src/main/java/org/apache/hadoop/hbase/util/VersionInfo.java
index aadad2e..8061b4d 100644
--- a/hbase-common/src/main/java/org/apache/hadoop/hbase/util/VersionInfo.java
+++ b/hbase-common/src/main/java/org/apache/hadoop/hbase/util/VersionInfo.java
@@ -19,6 +19,7 @@
 package org.apache.hadoop.hbase.util;
 
 import org.apache.commons.logging.LogFactory;
+import java.io.PrintStream;
 import java.io.PrintWriter;
 
 import org.apache.hadoop.hbase.classification.InterfaceAudience;
@@ -113,6 +114,12 @@ public class VersionInfo {
     }
   }
 
+  public static void writeTo(PrintStream out) {
+    for (String line : versionReport()) {
+      out.println(line);
+    }
+  }
+
   public static void logVersion() {
     for (String line : versionReport()) {
       LOG.info(line);
@@ -120,6 +127,6 @@ public class VersionInfo {
   }
 
   public static void main(String[] args) {
-    logVersion();
+    writeTo(System.out);
   }
 }
-- 
1.7.9.5

